automobile/scenario-creation-tutorial.md

```diff
@@ -68,7 +68,38 @@ To do this, open the `sumo.net.xml` file in SUMO `netedit`:
 $WEBOTS_HOME/projects/automobile/resources/sumo/bin/netedit $WBT_PROJECT_PATH/worlds/myMap_net/sumo.net.xml
 ```
 
-You can then generate the route file, SUMO provides several ways to generate route files. You may for example generate a [flow file](http://sumo.dlr.de/wiki/Definition_of_Vehicles,_Vehicle_Types,_and_Routes) and then use [duarouter](http://sumo.dlr.de/wiki/DUAROUTER) to generate the route file for you:
+
+### Generate the SUMO route file
+
+At this stage, a SUMO route file should be created to describe the traffic, i.e.,
+the number of vehicles, which road they should take, the vehicle properties, etc.
+
+This [SUMO document](http://sumo.dlr.de/wiki/Definition_of_Vehicles,_Vehicle_Types,_and_Routes) explains precisely how to achieve this.
+
+This following subsections describes two ways to create a SUMO route file.
+The use of the SUMO [duarouter](http://sumo.dlr.de/wiki/DUAROUTER) program is required in both cases.
+
+
+#### Create randomly a SUMO route file
```

Comment:
I'd move "randomly" to then end of this heading e.g. "Create a SUMO route file randomly , same for the following heading.